src/actions/guides/getting-started/installing.cr

```diff
@@ -31,11 +31,15 @@ class Guides::GettingStarted::Installing < GuideAction
     to your `~/.bash_profile` or `~/.zshrc` so Crystal can use SSL through the
     openssl package.
 
-    ```shell
-    # For Bash, the default shell on MacOS
+    **For Bash, the default shell on MacOS:**
```

Comment:
Default is now `zsh` if you have a new mac with Catalina. I think it's only bash if you upgraded to Catalina. [ref](https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features)

Reply:
Maybe just say `For Bash` and `For ZSH`, and then a note that if you're unsure, you can `echo $SHELL` ?

Reply:
This is great to know! I’ll definitely change this and mention the echo $SHELL to determine which one